yarn.lock

```diff
@@ -387,6 +387,52 @@
     tslib "^2.5.0"
     uuid "^8.3.2"
 
+"@aws-sdk/client-codepipeline@^3.445.0":
```

Comment:
I think this just did not get updated to the fixed version.

Reply:
Updated!